src/promptflow-azure/promptflow/azure/operations/_artifact_utilities.py

```diff
@@ -103,6 +110,27 @@ def get_datastore_info(operations: DatastoreOperations, name: str) -> Dict[str,
     return datastore_info
 
 
+def check_scope_rbac_role(client, scope: str, role_name: str = "Storage Blob Data Contributor"):
```

Comment:
why do we need to do such check? looks like we need to hard code a role name here. 
User can make other roles wich cover this capability?